Rocket.Chat/Controllers/Chat/MessagesViewModel.swift

```diff
@@ -184,8 +184,10 @@ final class MessagesViewModel {
      messages of the subscription attached to the view model.
      */
     internal func unsubscribe(for subscription: Subscription) {
```

Comment:
subscription parameter isn't being used anymore? Should we remove it and also in the subscribe method?